src/components/ui/LocalizedTime.js

```diff
@@ -2,38 +2,51 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
+import memoize from 'memoize-one';
 import moment from 'moment';
 import { isServerSide } from './../util/misc';
 
 
-export class LocalizedTime extends React.PureComponent {
+export class LocalizedTime extends React.Component {
 
     constructor(props){
         super(props);
-        this.state = {
-            moment : props.momentDate ? props.momentDate :
-                props.timestamp ? moment.utc(props.timestamp) : moment.utc(),
-            mounted : false
+        this.memoized = {
```

Comment:
This returns new moment instance if a new timestamp occurs. Previously, if timestamp changed, this wouldn't update in response.